docs/DATA_TYPES.md

```diff
@@ -0,0 +1,34 @@
+# TV Kitchen Data Types
+
+The [dataTypes.js](src/constants/dataTypes.js) constant file contains the list of standard types of data that the TV Kitchen and its appliances can produce.
+
+Someone creating a new appliance **is not bound to this list** and can invent new data types that their network of appliances might work with, it just won't be considered a standard data type unless it appears in this constants list.
+
+To suggest a new data type please open a [discussion issue]() in the base repository.
+
+## Naming Data Types
+
+Data type constants have some naming (and value) rules.
+
+* Data type buckets must have an `ALL` attribute.
+* Data type bucket names can only consist of `A-Za-z0-9` and `_`.
+* Data type bucket names cannot be longer than 16 characters.
+* Data type constant names can only consist of `A-Za-z0-9` and `_`.
+* Data type constant names cannot be longer than 32 characters.
+* Data type constant values must match the following pattern: `{BUCKETNAME}.{CONSTANTNAME}`.
+
+## Data type descriptions
+
+### STREAM
+
+STREAM content is generally pulled from an Ingestion Engine and passed to the countertop.
+
+* `STREAM.ALL` - Any type of stream data.
+* `STREAM.DATA` - Buffers representing a portion of data stream.
+
+### TEXT
+* `TEXT.ALL` - Any type of text data.
+* `TEXT.ATOM` - The smallest possible chunk of text (e.g. a character).
```

Comment:
_Should_ this just be `TEXT.CHARACTER`? Is there any time an atom won't be a character?

Reply:
I wasn't sure if we might face special types of... characters? like an "EOL" or equivalent (or like, color signals, or WHO EVEN KNOWS... EMOJI?)

Given that, what do you thin?  I'm open to CHARACTER if you think that would still be appropriate for those other things.

Reply:
I say we lean into this. `ATOM` can be destructured to `PARTICLE` and perhaps `QUARK`?!

So I just typed up like three paragraphs and deleted them. I don't think the current list is perfect, but I do think we'll end up with something like `ATOM` despite any other types we decide to add. So it's a good start.

Reply:
There was a bit of back and forth thinking about this in slack -- I think in the spirit of time boxing and moving forward we should move with ATOM.  *however* I also think that CHARACTER might be the right name.

We're gonna move with ATOM for right now but circle back after we've tried using this.